sdcm/nemesis.py

```diff
@@ -1022,7 +1022,19 @@ def _choose_file_for_destroy(self, ks_cfs):
 
     @decorate_with_context(ignore_ycsb_connection_refused)
     def _destroy_data_and_restart_scylla(self):
-
+        def find_five_files_to_destroy():
+            all_files_to_destroy = []
+            try:
+                for _ in range(5):
```

Comment:
why do we need double loop here ? it's really not clear

Reply:
There's a chance that `_choose_file_for_destroy` will pick a file that is already in the list. The second loop is a retry/re-raffle.

Reply:
to avoid duplication I would use something like `set`

Reply:
It's a retry loop.

Reply:
It's just not very readable, since he use the break if it's not in the list.

Reply:
but I think it's good enough.

Reply:
I'm still not following what this code is trying to solve, we have better ways of retries.

And I don't understand which problem it's trying to solve.

Reply:
I know we have better ways, I just explained what he tries to do here.
The PR is trying to "solve" or "ease" the requirement of 5 files to be destroyed, so we won't fail the test with ERROR in case we found less than 5 files.